spec/index.ts

```diff
@@ -1,8 +1,3 @@
-import chai from 'chai';
-import { solidity } from 'ethereum-waffle';
-
-chai.use(solidity);
```

Comment:
Not sure why this was needed.  Is it possible that removing it will break some SS installations?

Reply:
I apologize for the delay in my response. 

Looks like @Lumyo ran into an issue in dda39ac. I didn't experience any issues building and as far as usage, it would be expected that the user of this repo's spec package would be using the `@nomicfoundation/hardhat-chai-matchers` package. hardhat-chai-matchers in turn does this setup [here](https://github.com/NomicFoundation/hardhat/blob/09ebee4aef831392ec5b5cce022d7f03fbd8c3aa/packages/hardhat-waffle/src/waffle-chai.ts).

This is therefore a breaking change for the spec package.

Reply:
I think the solution is to make the matchers package a peer dependency.  Though I haven't worked with peer dependencies much, and don't understand all of the implications.